docs/reference/go/go.md

```diff
@@ -0,0 +1,194 @@
+---
+bodyclass: docs
+headline: Go Generated Code Guide
+layout: docs
+type: markdown
+---
+
+<p class="lead">This guide gives an overview of code generated with the protoc grpc go plugin.</p>
+
+<div id="toc"></div>
+
+## Compiler Invocation and Results of Code Generation
+This section provides instructions for using the [grpc plugin](https://godoc.org/github.com/golang/protobuf/protoc-gen-go/grpc) to `protoc-gen-go`, when compiling `.proto` files with `protoc`.
```

Comment:
This section doesn't really give you instructions for that, it just points you at some other docs.

I'd change it to:

This guide describes the code generated with the [grpc plugin](https://godoc.org/github.com/golang/protobuf/protoc-gen-go/grpc) to `protoc-gen-go` when compiling `.proto` files with `protoc`.